name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/PDFParser.java

```diff
@@ -209,6 +242,23 @@ public Section(Transaction<T> transaction, String[] attributes)
             return this;
         }
 
+        public Section<T> exclusiveOrGroup(int groupID)
+        {
+            this.isGroup = true;
+            this.groupID = groupID;
+            return this;
+        }
+        
+        public boolean IsGroup()
```

Comment:
Die Namenskonvention für Java Methoden ist: mit einem Kleinbuchstaben anzufangen. Dann CamelCase. Ist nur eine Kleinigkeit, aber aus die Kleinigkeiten sollten stimmen.

Reply:
Komme aus der C-Welt und bin daher PascalCase für public Methoden und lowerCamelCase für private Geschichten gewöhnt. Aber ich kann hier gerne auch alles in lowerCamelCase machen bzw. sehe ich gerade das ich eh nicht konsistent war. Habe mir gerade die Erweiterung CheckStyle geladen ums nicht wieder zu vergessen ;)

Reply:
Kein grosses Problem. BTW, ich habe bei mir https://www.sonarlint.org in Eclipse installiert. Der meldet zwar auch einige "Probleme" die ich nicht unbedingt immer als Probleme ansehe, aber hilft auch solche Kleinigkeiten zu sehen. (Und ich will auch nicht als Korinthenkacker rüberkommen - die eine Methode ist nicht schlimm, es ist eher wenn sich sowas addiert).